main.js

```diff
@@ -50,9 +63,141 @@ function countPrefPerClass(data) {
             }
         });
     });
+
+    //Alert & Remove classes without enough preferences to make up the minimum requiredxs
+    let insufficientClasses = []
+    for(let thisClass in classes) {
+        if(classes[thisClass[0]] < minMentor){
+            insufficientClasses.push(thisClass);
+            delete classes[thisClass];
+            classCount--;
+        }
+    }
+    if(insufficientClasses.length>0) alert(`The folowing classes have insufficient\
+    preferences and could not be allocated:\n${insufficientClasses}`);
+}
+
+function countExperiencedMentors(data) {
+    let numExperienced = 0;
+    data.forEach(function(mentor) {
+        if(mentor["Have you mentored before?"]){
+            numExperienced++;
+        }
+    });
+    console.log(`Number of experience mentors: ${numExperienced}`);
 }
 
 // Handles errors in parsing CSV file
 function errorFn() {
 
+}
+
+function bruteForce(data) {
+    let conditionsSatisfied = false;
+
+    let iterationCounter = 0;
+    let classAllocations;
+    while(!conditionsSatisfied) {
+        classAllocations = assignRandomMentors(data);
+        let sizeSatisfied = checkSizes(classAllocations);
+        let driversSatisfied = checkAtLeastOneDriver(classAllocations);
+        let experiencedSatisfied = checkExperienced(classAllocations);
+        conditionsSatisfied = (sizeSatisfied && driversSatisfied && experiencedSatisfied) ? true : false;
```

Comment:
Had a closer looks works great, probably some refactoring could be done but it doesn't really matter either way at this stage